CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.0.6] - TBA
```

Comment:
2.0.6 is the previous version number. I think it's easier to leave this blank or set it to TBD and only fill it in just before doing a release.